Libs/vtkSegmentationCore/vtkSegmentationModifier.cxx

```diff
@@ -198,6 +198,25 @@ void vtkSegmentationModifier::SeparateModifiedSegmentFromSharedLabelmap(vtkOrien
     }
 }
 
+inline int* GetMaximumExtent(const int *extent, vtkOrientedImageData* labelmap)
```

Comment:
I think the name `GetMaximumExtent` is a bit misleading.
Technically, what we want is the "union" of the 2 extents.

Maybe rename it to `GetExtentUnion` (and add a comment that it works only if `extent` is ensured to be within labelmap extent)?

Reply:
Modified the name and added a docstring specifying the method's limitations